src/services/riotApiClient.test.js

```diff
@@ -3,13 +3,18 @@ import mockAxios from 'axios'
 import riotApiClient from './riotApiClient'
 import config from 'config'
 
+const log = require('loglevel')
+beforeAll(() => {
+  log.disableAll()
```

Comment:
You can probably get rid of this as well. We actually don't want to turn logging off for tests.